portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -15,17 +18,25 @@
 
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-
+        String lanString = "es";
```

Comment:
This should be a `private final static String` in the class so that it is not reallocated every time doPost is called.